pkg/rekor/identity_test.go

```diff
@@ -717,3 +850,181 @@ func TestOIDMatchesValue(t *testing.T) {
+		ObjectIdentifier: asn1.ObjectIdentifier{2, 5, 29, 17},
+		ExtensionValues:  []string{"test1"},
+	}, {
+		ObjectIdentifier: asn1.ObjectIdentifier{2, 5, 29, 18},
+		ExtensionValues:  []string{"test"},
+	}}})
+	if err != nil {
+		t.Errorf("Expected nil, got %v", err)
+	}
+	if len(oidMatchers) != 2 {
+		t.Errorf("Expected 1 OIDMatcher, got %d", len(oidMatchers))
+	}
+}
+
+// test parseObjectIdentifier
+func TestParseObjectIdentifier(t *testing.T) {
+	oid, err := parseObjectIdentifier("")
+	if err == nil {
+		t.Errorf("Expected error, got nil and oid %s", oid)
+	}
+
+	oid, err = parseObjectIdentifier(".")
+	if err == nil {
+		t.Errorf("Expected error, got nil and oid %s", oid)
+	}
+
+	oid, err = parseObjectIdentifier("....")
+	if err == nil {
+		t.Errorf("Expected error, got nil and oid %s", oid)
+	}
+
+	oid, err = parseObjectIdentifier("a.a")
+	if err == nil {
+		t.Errorf("Expected error, got nil and oid %s", oid)
+	}
+
+	oid, err = parseObjectIdentifier("1.")
+	if err == nil {
+		t.Errorf("Expected error, got nil and oid %s", oid)
+	}
+
+	oid, err = parseObjectIdentifier("1.1.5.6.7.8..")
+	if err == nil {
+		t.Errorf("Expected error, got nil and oid %s", oid)
+	}
+
+	oid, err = parseObjectIdentifier(".1.1.5.67.8")
+	if err == nil {
+		t.Errorf("Expected error, got nil and oid %s", oid)
+	}
+
+	_, err = parseObjectIdentifier("1")
+	if err != nil {
+		t.Errorf("Expected nil, got error %v", err)
+	}
+
+	_, err = parseObjectIdentifier("1.4.1.5")
+	if err != nil {
+		t.Errorf("Expected nil, got error %v", err)
+	}
+
+	_, err = parseObjectIdentifier("11254215212.4.123.54.1.622")
+	if err != nil {
+		t.Errorf("Expected nil, got error %v", err)
+	}
+}
+
+// test renderFulcioOIDMatchers
+func TestRenderFulcioOIDMatchers(t *testing.T) {
+	extValueString := "test cert value"
+	fulcioExtensions := extensions.FulcioExtensions{
+		Issuer:                              []string{},
+		GithubWorkflowTrigger:               []string{},
+		GithubWorkflowSHA:                   []string{},
+		GithubWorkflowName:                  []string{},
+		GithubWorkflowRepository:            []string{},
+		GithubWorkflowRef:                   []string{},
+		BuildSignerURI:                      []string{extValueString},
+		BuildSignerDigest:                   []string{},
+		RunnerEnvironment:                   []string{},
+		SourceRepositoryURI:                 []string{},
+		SourceRepositoryDigest:              []string{},
+		SourceRepositoryRef:                 []string{},
+		SourceRepositoryIdentifier:          []string{},
+		SourceRepositoryOwnerURI:            []string{},
+		SourceRepositoryOwnerIdentifier:     []string{},
+		BuildConfigURI:                      []string{"1", "2", "3", "4", "5", "6"},
+		BuildConfigDigest:                   []string{},
+		BuildTrigger:                        []string{},
+		RunInvocationURI:                    []string{},
+		SourceRepositoryVisibilityAtSigning: []string{},
```

Comment:
I think the default value is `[]string{}` so we don't need to include those fields.

https://go.dev/play/p/xvFuPti3B9F